arelle/Cntlr.py

```diff
@@ -179,12 +179,10 @@ def __init__(self, hasGui=False, logFileName=None, logFileMode=None, logFileEnco
                 self.userAppDir = os.path.join( impliedAppDir, "Arelle")
             if hasGui:
                 try:
-                    import win32clipboard
```

Comment:
Not sure about this one and the other ones below, feels like they actually need to be imported?